utils/spacewalk-common-channels.ini

```
@@ -3179,3 +3179,51 @@ gpgkey_url = http://download.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
 gpgkey_id = 2F86D6A1
 gpgkey_fingerprint = 94E2 79EB 8D8F 25B2 1810 ADF1 21EA 45AB 2F86 D6A1
 repo_url = http://mirrors.fedoraproject.org/mirrorlist?repo=epel-8&arch=%(arch)s
+
+[openeuler2203-os]
```

```suggestion
[openeuler2203]
```
The parent channel should have a "neutral" name, so the children can inherit.